README.md

```diff
@@ -28,6 +28,9 @@ More complete install documentation is available on the PostgreSQL [wiki](https:
 
 **Note:** If you're setting up the project on Mac OSX, we recommend using [homebrew](http://brew.sh) to install PostgreSQL.
 
+### Install Memcached
```

Comment:
Ah--thanks for the info. Do we need pylibmc for local development? I'd prefer not to make local devs have to install memcached. Heck, I'd prefer not to install it. We did something similar to @catherinedevlin's suggestion on the broker (2 separate requirements.txt files).